### PR TITLE
Grafana units

### PR DESCRIPTION
**Pull request by dbrinegar**
Introduces usage of Grafana keys for `legendFormat` and `yaxes` which control the labeling under the graph and the terms and units on y-axis respectively.

fixes lightbend/es-backend#378

### CODE REVIEW

**Comment by dbrinegar** [comment]
oh! must teach the yaml validator the new format...

File: enterprise-suite/es-grafana/akka-exporter.json [hunk 1]
```diff
@@ -1,14 +1,138 @@
...
-  {"graphName":"Unhandled Messages Rate", "promQL":["sum by (actor)(irate(akka_actor_unhandled_message{ContextTags}[1m]))"]},
-  {"graphName":"Dead Letters Rate", "promQL":["sum by (actor) (irate(akka_actor_dead_letter{ContextTags}[1m]))"]},
-  {"graphName":"Dispatcher Queue Size", "promQL":["max by (actor_system) (akka_dispatcher_queue_size{ContextTags})"]},
-  {"graphName":"Dispatcher Pool Size", "promQL":["sum by (actor_system) (akka_dispatcher_pool_size{ContextTags})"]},
-  {"graphName":"Dispatcher Threads", "promQL":["sum by (actor_system) (akka_dispatcher_active_threads{ContextTags})","sum by (actor_system) (akka_dispatcher_running_threads{ContextTags})"]}
+  {
+    "graphName": "Mailbox Size",
+    "promQL": [
+      {
+        "expr": "max by (actor)(akka_actor_mailbox_size{ContextTags,quantile=\"0.99\"})",
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/akka-exporter.json]: No quantile bit here before.  Was adding it intentional?

File: enterprise-suite/es-grafana/akka-exporter.json [hunk 2]
```diff
@@ -1,14 +1,138 @@
...
+  {
+    "graphName": "Processing Time",
+    "promQL": [
+      {
+        "expr": "max by (actor)(akka_actor_processing_time_ns{ContextTags,quantile=\"0.99\"})",
+        "legendFormat": "{{actor}} max"
+      }
+    ],
+    "yaxes": {
+      "format": "ns"
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/akka-exporter.json]: You used `"label": "ns"` in akka-http-exporter.json below.  Should those be consistent?

File: enterprise-suite/es-grafana/akka-exporter.json (hunk 1 above)
Comment (dbrinegar) [reply enterprise-suite/es-grafana/akka-exporter.json]: yes, leaving it out is a bug. max of all quantiles is ... non-sense

File: enterprise-suite/es-grafana/akka-exporter.json (hunk 2 above)
Comment (dbrinegar) [reply enterprise-suite/es-grafana/akka-exporter.json]: frack. good catch

File: enterprise-suite/es-grafana/play-exporter.json [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+      "sum by (http_client)(irate(play_http_client_play_client_requests{ContextTags}[1m]))"
+    ]
+  },
+  {
+    "graphName": "Play Client Response Time",
+    "promQL": [
+      "max by (http_client)(play_http_client_play_client_service_response_time_ns{ContextTags, quantile=\"0.99\"})"
+    ],
+    "yaxes": {
+      "label": "ns"
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/play-exporter.json]: This one too then?

**Pull request merged**